eluthia/build.py

```diff
@@ -56,3 +57,8 @@ def build_app(app):
         for path, f in flatten_lists(flatten_paths(tree)):
             full_path = (build_folder, package_name, *path)
             f(full_path, *args)
+        
+        # I'm not sure if nginx-conf is meant to be packaged, but it doesn't have a control file so it can't be. This ensures that
+        # it isn't packaged, which would cause an error.
+        if package_name in apps.config.keys():
+            subprocess.run(["dpkg-deb", "--build", f"{build_folder}/{package_name}"],check=True)
```

Comment:
i also want to run this script without dpkg-deb, because I also use ArchLinux. Provide an environment variable to skip this step. eg. SKIP_DEB=yes